Plugins/ConsolePanel/Settings.cs

```diff
@@ -34,5 +40,51 @@ public ConsoleColor ForegroundColor
                 this.foreground = value;
             }
         }
+
+        [DisplayName("Terminal"), DefaultValue(TerminalProvider.Bash)]
+        public TerminalProvider TerminalProvider
+        {
+            get
+            {
+                return this.provider;
+            }
+            set
+            {
+                this.provider = value;
+            }
+        }
+
+        [Category("Bash"), DisplayName("Command"), DefaultValue(@"C:\Program Files\Git\usr\bin\mintty.exe")]
+        [Editor(typeof(FileNameEditor), typeof(UITypeEditor))]
+        public string MinttyCommand
+        {
+            get
+            {
+                return this.minttyCmd;
+            }
+            set
+            {
+                this.minttyCmd = value;
+            }
+        }
+
+        [Category("Bash"), DisplayName("Parameters"), DefaultValue("-e \"/c/Program Files/Git/bin/bash\"")]
```

Comment:
I cannot do it for this specific line. As you can see it is not standard command line format. It cannot contain the ":". However, I will do it for the other one.